### ISSUE TITLE
Return values

### ISSUE THREAD

**Issue by edemaine** [peerlibrary/meteor-reactive-publish#9]
Just discovered this great package which is perfect for subscribing to things a user has permission to access!

A small issue I encountered because I use CoffeeScript: the following code does not work.

```
Meteor.publish 'items', ->
  @autorun ->
    Items.find
      group: $in: Meteor.users.findOne(@userId).groups
```

The reason is that `this.autorun` returns something (a computation?), and in CoffeeScript, this gets automatically returned to `Meteor.publish`, which then complains "Publish function can only return a Cursor or an array of Cursors".

Not a big deal.  I'm not sure whether you can/should modify `Meteor.publish` to allow another type of return value for this common case.  But at least it might not hurt to mention this (common?) caveat for CoffeeScript users in the README.

**Comment by mitar** [comment]
Yes, I also do not know what would be the best to do here.

**Comment by hexsprite** [comment]
argh just got caught with this, yes a note in the docs would help and also an example of how to fix it

**Comment by hexsprite** [comment]
ok so the fix is just to return `null` after defining your autorun...

**Comment by mitar** [comment]
Or just `return`, without `null`.

**Comment by mitar** [comment]
Fixed. Now you can return computation. And it will be even automatically stopped for you.

Thanks for reporting.

**Comment by mitar** [comment]
Available in 0.2.0.

**Comment by edemaine** [comment]
Works great; thanks!
